__tests__/reducers/organization.test.ts

```diff
@@ -45,7 +45,7 @@ describe('Organization Reducer', () => {
 		expect(expectedState).to.deep.equal(actualState)
 	})
 
-	it('should handle UPDATED_ORGANIZATION.', () => {
+	it('should handle UPDATED_ORGANIZATION.', async () => {
```

Comment:
@misstonbon either all of these need to follow a async/await pattern or none. Grab me if this is confusing